examples/rust/tcp-ping-pong.rs

```diff
@@ -261,12 +266,15 @@ fn client(remote: SocketAddrV4) -> Result<()> {
             }
         }
 
+        let mut fill_check: u8 = (i % (u8::MAX as usize - 1) + 1) as u8;
+
         // Pop data, and sanity check it.
         {
             let mut recvbuf: [u8; BUFFER_SIZE] = [0; BUFFER_SIZE];
             pop_and_wait(&mut libos, sockqd, &mut recvbuf);
             for x in &recvbuf {
-                assert!(*x == fill_char);
+                assert!(*x == fill_check);
```

Comment:
Could you change this to `assert_eq()`, this would print the values if it fails?

Reply:
Ah perfect